doc/contributing/submitting-changes.chapter.md

```diff
@@ -234,6 +234,15 @@ digraph {
 
 The `master` branch is the main development branch. It should only see non-breaking commits that do not cause mass rebuilds.
 
+The guideline for proposed change is good for `master` branch:
+
+- Rebuild count is less than 1000 dependent 1000.
+- Rebuild count is between 1000 and 2000 lightweight packages (like python depends) and does not contain heavyweight package rebuilds (like `chromium` or `firefox`).
```

Comment:
2000 python rebuilds should probably also go to staging to not overwhelm Hydra. It is a rule of thumb. If the update is important it is easier to justify that it goes to master. If it is not important at all it should go to staging regardless.

Also it should be considered if the package update is likely to break depending packages. If so we should target staging to not break hundreds or thousands of packages.

Reply:
Decreased the cutoff limit to `1000` and mentioned simple leaf non-risky package bumps separately. Added more words on the intent not to overload Hydra.